docs/what-we-look-at.md

```diff
@@ -0,0 +1,48 @@
+# What we look at
+
+Licensee works by taking a detected license file, and comparing the contents to a short list of known licenses.
+
+## Detecting the license file
+
+Licensee uses [a series of regular expressions](https://github.com/benbalter/licensee/blob/master/lib/licensee/project_files/license_file.rb#L15-L25) to score files in the project's root as potential license files. Here's a few examples of files that would be detected:
+
+* `LICENSE`
+* `LICENCE`
+* `license.md`
+* `COPYING.txt`
+* `LICENSE-MIT`
+* `COPYRIGHT`
+* `UNLICENSE`
+
+## Known licenses
+
+Licensee relies on the crowdsourced license content and metadata from [`choosealicense.com`](http://choosealicense.com).
+
+## What it doesn't look at
+
+* The licensing of a project's dependencies
+* References to licenses in `README`, `README.md`, etc.
+* Every single possible license (just the most popular ones)
+* Compliance - If you're looking for this, take a look at [LicenseFinder](https://github.com/pivotal/LicenseFinder).
+
+## Huh? Why don't you look at X?
+
+Because reasons.
+
+### Why not just look at the "license" field of [insert package manager here]?
+
+Because it's not legally binding. A license is a legal contract. You give up certain rights (e.g., the right to sue the author) in exchange for the right to use the software.
+
+Most popular licenses today *require* that the license itself be distributed along side the software. Simply putting the letters "MIT" or "GPL" in a configuration file doesn't really meet that requirement.
+
+Not to mention, it doesn't tell you much about your rights as a user. Is it GPLv2? GPLv2 or later? Those files are designed to be read by computers (who can't enter into contracts), not humans (who can). It's great metadata, but that's about it.
```

Comment:
I foresee a future FAQ entry about using SPDX identifiers instead in package manager metadata files ;)